zombienet_tests/misc/0001-paritydb.zndsl

```diff
@@ -42,16 +42,16 @@ validator-0: parachain 2008 is registered
 validator-0: parachain 2009 is registered
 
 # Check if network is fully connected.
-validator-0: reports peers count is at least 19 within 20 seconds
-validator-1: reports peers count is at least 19 within 20 seconds
-validator-2: reports peers count is at least 19 within 20 seconds
-validator-3: reports peers count is at least 19 within 20 seconds
-validator-4: reports peers count is at least 19 within 20 seconds
-validator-5: reports peers count is at least 19 within 20 seconds
-validator-6: reports peers count is at least 19 within 20 seconds
-validator-7: reports peers count is at least 19 within 20 seconds
-validator-8: reports peers count is at least 19 within 20 seconds
-validator-9: reports peers count is at least 19 within 20 seconds
+validator-0: reports peers count is at least 8 within 20 seconds
```

Comment:
This I'm not entirely sure about because it's I'm not sure how it was previously determined